### ISSUE TITLE
Formatting Button Error

### ISSUE THREAD

**Issue by anmode** [accordproject/web-components#364]
A big error is thrown when there is a click over the bulleted and numbered list button.
# Bug Report 🐛
Error: Cannot find a descendant at path [1,0,0,6] in node.
This error with a big message is shown when we click over the bulleted or numbered button. 
This error comes in markdown editor only.

## Expected Behavior
It should just add a bullet and number.

## Current Behavior
Now a big error message is thrown on the editor.

## Possible Solution
I think there is a problem with the js.
http://localhost:9009/vendors~main.iframe.bundle.js
![Screenshot from 2022-01-11 13-42-42](https://user-images.githubusercontent.com/88332977/148963837-85047e0e-ec9d-498e-a056-d05e7ebed4ab.png)

## Steps to Reproduce

1. Open Markdown editor.
2. Take a custom paragraph
3. Then click over the bulleted or numbered button
4. A error will be thrown

## Context (Environment)
 - OS: Ubuntu 21.10
 - Browser: Brave
 - Version: 1.32.113 Chromium: 96.0.4664.45 (Official Build) (64-bit)

**Comment by anmode** [comment]
Due to this bug I am unable to solve #284 bug.

**Comment by irmerk** [comment]
Yes I see this error. Unsure at the moment what the cause is, and it will take some investigation.

**Comment by AbhijithGanesh** [comment]
@irmerk Is this issue reserved for existing contributors or as a first-timer can I contribute to this? If so, can I be assigned to this?

**Comment by irmerk** [comment]
This issue is affecting more buttons than just the list buttons, and likely has the same cause. 

@AbhijithGanesh feel free to work on it.

**Comment by anmode** [comment]
@irmerk i m also working on this issue...yeah u are correct this issue is affecting more button in the markdown editor

**Comment by mansigrover09** [comment]
Hey, i want to work on this issue. Can you guide me regarding the same?

**Comment by mttrbrts** [comment]
@mansigrover09 you're welcome to contribute, although it doesn't appear that anyone has been able to find a root cause yet.

**Comment by pavankumarraju1** [comment]
I want to work on this issue. May i know the update on this issue?

**Comment by mttrbrts** [comment]
No-one has yet found the route cause, as far as I know. You're welcome to pick this up @pavankumarraju1

**Comment by Ayush1404** [comment]
@mttrbrts Is this still an issue ? I think it might have been solved as i am able to click on all the buttons in markdown now , and everything seems to be working fine

**Comment by Tanaykmr** [comment]
@mttrbrts seems like this issue should be closed, it must have gotten fixed somewhere along the way.

**Comment by sanketshevkar** [comment]
Idts that we merged any changes related to this. There are specifics environrmnts where people encounter this.

**Comment by Tanaykmr** [comment]
weird, works fine on chrome Chrome 131.0.6778.205(Official Build) with MacOS 15.0.01

**Comment by sanketshevkar** [comment]
We no longer maintain this repo. Thanks for showing your interest. If you're intereseted in contributing to Accord Project. Please check out our other repos.

template-playground
concerto packages
